src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -28,6 +32,11 @@ public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
 
+    /**
+     * Formatter for String to LocalDateTime.
+     */
+    private static DateTimeFormatter strFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
```

Comment:
We might want to have a separate format for different recurrence levels.

Daily: Time
Weekly: Time and Day of the week
Monthly: Time and Day of the month
Non-recurring: year, month, day  and time